tests/UI/pages/FO/home/index.js

```diff
@@ -320,6 +328,19 @@ class Home extends FOBasePage {
 
     return this.openLinkWithTargetBlank(page, selector, 'body');
   }
+
+  /**
+   * subscribe to the newsletter from FO homepage
+   * @param page
+   * @param email
```

Comment:
```suggestion
   * @param {object} page
   * @param {string} email
```

At least, or maybe page has a type?